site/_includes/layouts/blank.njk

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
Please add comment of why this template exists and where is used. Or maybe consider renaming to test.njk or similar?

Reply:
Renamed to test and added a comment header.